cli/cdsctl/admin_infos.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"reflect"
+
+	"github.com/ovh/cds/cli"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	adminInfosCmd = cli.Command{
+		Name:  "infos",
```

Comment:
there is no plural form of info:
https://www.linguee.com/english-french/translation/info.html

btw i don't think it's the right word, what's the goal ?

Reply:
will be broadcast in next commit